.github/workflows/continous-integration-docker.yml

```diff
@@ -470,14 +470,14 @@ jobs:
   #
   # If the state version has changed, waits for the new cached state to be created.
   # Otherwise, if the state rebuild was skipped, runs immediately after the build job.
-  generate-checkpoints-mainnet:
+  checkpoints-mainnet:
     name: Generate checkpoints mainnet
     needs: [ test-full-sync, get-available-disks ]
     uses: ./.github/workflows/deploy-gcp-tests.yml
     if: ${{ !cancelled() && !failure() && (fromJSON(needs.get-available-disks.outputs.zebra_tip_disk) || needs.test-full-sync.result == 'success') && github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' && github.event.inputs.run-lwd-sync != 'true' }}
     with:
       app_name: zebrad
-      test_id: generate-checkpoints-mainnet
+      test_id: checkpoints-mainnet
```

Comment:
Changing this test ID will also change the name of the Run job, so we'll need to update the branch protection rules. Let's do that after most PRs have merged.

The other ID changes are for scheduled or testnet jobs, so they aren't in the branch protection rules.